src/physics/euler.h

```diff
@@ -206,11 +206,16 @@ class Euler : public PhysicsBase <dim, nstate, real>
     /// Opposite of convert_primitive_to_conservative
     std::array<real,nstate> convert_primitive_to_conservative ( const std::array<real,nstate> &primitive_soln ) const;
 
+    /// Given conservative variables [density, [momentum], total energy],
+    /// returns entropy variables [(gamma-entropy)/(gamma-1)-0.*density*vel2/pressure, [momentum/pressure], -density/pressure]
```

Comment:
maybe just add the reference for this

Reply:
Sure, I've left a reference. The actual vector of variables is clear from the code if other users want to read into it.